resources/functionMap.php

```diff
@@ -11546,7 +11546,8 @@
 'sqlsrv_server_info' => ['array', 'conn'=>'resource'],
 'sqrt' => ['float', 'number'=>'float'],
 'srand' => ['void', 'seed='=>'int', 'mode='=>'int'],
-'sscanf' => ['mixed', 'str'=>'string', 'format'=>'string', '&...w_vars='=>'string|int|float|null'],
+'sscanf' => ['int|null', 'str'=>'string', 'format'=>'string', '&...w_vars'=>'string|int|float|null'],
```

Comment:
@ondrejmirtes I have no idea why the tests fails, can you help me?
I made variadic args not optional, but I still get `array|int|null` for two-arguments signature, but I want just `array|null`.

Reply:
This is why I wanted you to write these tests because I wasn't sure it's going to work the way you want. You need to modify the signature so that there's a third required parameter (non-variadic), and the fourth one is variadic and optional.